### PR TITLE
test(switchMapFirst): expand test coverage for switchMapFirst

### PR DESCRIPTION
**Pull request by kwonoj**
This PR adds couple of tests to cover `switchMapFirst`.

### CODE REVIEW

File: spec/operators/switchMapFirst-spec.js [hunk 1]
```diff
@@ -33,6 +33,31 @@ describe('Observable.prototype.switchMapFirst()', function () {
     expectSubscriptions(x.subscriptions).toBe(xsubs);
   });
 
+  it('should raise error if project throws', function () {
+    var x = cold(     '--a--b--c--|                              ');
+    var e1 =   hot('---x---------y-----------------z-------------|');
+    var expected = '---#';
```
Comment (staltz) [inline spec/operators/switchMapFirst-spec.js]: Good stuff, but can you add also an unsubscription assertion here for the error raised in this test case?

**Comment by kwonoj** [comment]
Updated PR to add unsubscription tests.

**Comment by staltz** [comment]
Looks good to me(rge).

**Comment by kwonoj** [comment]
Merged with ecdd41b.

**Pull request closed without merging**